src/PIL/ImageGrab.py

```diff
@@ -93,12 +93,29 @@ def grabclipboard():
         os.unlink(filepath)
         return im
     elif sys.platform == "win32":
-        data = Image.core.grabclipboard_win32()
+        fmt, data = Image.core.grabclipboard_win32()
+        if fmt == "file":  # CF_HDROP
+            import struct
+
+            o = struct.unpack_from("I", data)[0]
+            if data[16] != 0:
+                files = data[o:].decode("utf-16le").split("\0")
+                return files[: files.index("")]
+            else:
+                files = data[o:].decode("mbcs").split("\0")
+                return files[: files.index("")]
```

Comment:
```suggestion
            else:
                files = data[o:].decode("mbcs").split("\0")
            return files[: files.index("")]
```
Replaced two identical lines with one.

Reply:
Good idea.